modules/admin_manual/pages/configuration/files/file_versioning.adoc

```diff
@@ -1,38 +1,127 @@
-* For the first hour ownCloud keeps one version every minute
-* For the first 24 hours ownCloud keeps one version every hour
-* For the first 30 days ownCloud keeps one version every day
-* After the first 30 days ownCloud keeps one version every week
+Every time when a file is written to the storage the versions app
+(files_versions) creates a new backup copy of the file. These backups are
+stored inside a folder `files_versions` inside the users root folder.
+The app will add the suffix `.v` followed by the unix timestamp of the creation
+date of the backup copy.
+
+....
+.
+├── files
+│   └── welcome.txt
+└── files_versions
+    ├── welcome.txt.v1556203470
+    ├── welcome.txt.v1556203501
+    └── welcome.txt.v1556203567
+....
+
+Versions are displayed in the WebUI in the details view in the right sidebar if
+you click on the file row in the file listing. You can restore the current file
+to one of the earlier backup copies in the list by clicking on the restore icon
+of the specific version.
+
+image::files-versions.png[File Versions in the WebUI]
+
+== How Versions are expired
+
+The versions app expires old file versions
+automatically to ensure that users do not exceed their storage quotas.
+This is done by automatic background jobs which clean up the versions following
+a specific pattern.
+
+==== This is the default pattern used to delete old versions:
+
+* For the last second we keep one version
+* For the last 10 seconds ownCloud keeps one version every 2 seconds
+* For the last minute ownCloud keeps one version every 10 seconds
+* For the last hour ownCloud keeps one version every minute
+* For the last 24 hours ownCloud keeps one version every hour
+* For the last 30 days ownCloud keeps one version every day
+* If the versions are older than 30 days ownCloud keeps one version every week
 
 The versions are adjusted along this pattern every time a new version is
-created.
+created and the background job has been run.
+
+=== Example
+
+[cols="1,1", options="header"]
+|===
+|Time Period before last Expiration
+|Maximum Number of Versions:
+
+|1 second
+|1
+
+|10 seconds
+|5
+
+| 1 minute
+| 6
+
+| 1 hour
+| 59
+
+| 1 day
+| 23
+
+| 30 days
+| 30
+
+|===
 
-The Versions app never uses more that 50% of the user’s storage quota.
+[WARNING]
+The versions app never uses more that 50% of the user’s storage quota.
 If the stored versions exceed this limit, ownCloud deletes the oldest
 file versions until it meets the disk space limit again.
 
+[TIP]
+Adjust the `'versions_retention_obligation'` setting in `config.php` to avoid
+filling up the user's quota.
+
+== Change the Expiration Settings
+
 You may alter the default pattern in `config.php`. The default setting
 is `auto`, which sets the default pattern:
 
 ....
 'versions_retention_obligation' => 'auto',
 ....
 
-Additional options are:
+Possible config values are:
 
 * `D, auto`: Keep versions at least for D days, apply expiration rules to all versions that are older than D days
```

Comment:
it might be worth converting this into a table which improves readbility, see also my column comment above

Reply:
👍